aten/src/ATen/DLConvertor.cpp

```diff
@@ -152,7 +152,7 @@ DLDevice torchDeviceToDLDevice(at::Device device) {
   return ctx;
 }
 
-static Device getATenDevice(DLDeviceType type, c10::DeviceIndex index, void* data = nullptr) {
+Device getATenDevice(DLDeviceType type, c10::DeviceIndex index, void* data) {
```

Comment:
If you removing a static, then I wonder if either namespace of function name should be change. `at::getATenDevice` feels like a very generic name. How about `at::dlpack::to_aten_device` (Function names usually follow snake case, don't they?)

Reply:
seems existing  API in DLConverter follows the style`TORCH_API DLDevice torchDeviceToDLDevice(at::Device device);`

maybe `TORCH_API torch::Device toATenDevice(DLDevice device);` ?